.github/workflows/macos_wheel.yaml

```diff
@@ -69,9 +99,30 @@ jobs:
 
           conda install "ffmpeg=${{ matrix.ffmpeg-version-for-tests }}" -c conda-forge
           ffmpeg -version
+
+      - name: Install test dependencies
```

Comment:
For this step I suggest we onboard to validation framework:
https://github.com/pytorch/test-infra/blob/main/.github/workflows/validate-domain-library.yml

This is exactly the use case we wrote validation framework for:
https://github.com/pytorch/test-infra/wiki/Validation-Framework